libraries/bot-applicationinsights/pom.xml

```diff
@@ -64,11 +64,23 @@
       <groupId>com.microsoft.bot</groupId>
       <artifactId>bot-builder</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.bot</groupId>
+      <artifactId>bot-dialogs</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.microsoft.bot</groupId>
+      <artifactId>bot-builder</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
```

Comment:
What's the difference between this and [this](https://github.com/southworks/botbuilder-java/pull/124/files#diff-80df2d58879ba1c659808f92a595950657b0dab09b5622907641a9bcbc113090R64)?

Confirm if we need this or not, it's weird.

Reply:
FYI, finally [this](https://github.com/southworks/botbuilder-java/pull/124/files#diff-80df2d58879ba1c659808f92a595950657b0dab09b5622907641a9bcbc113090R64) is not needed. Just leave this and it worked normally. Now we have the same as in [QnA library](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-ai-qna/pom.xml#L71).

Reply:
Great